osrf/vrx
# Switching to the USVWind plugin in the rest of the worlds

In PR #642 we updated the worlds in `practice_2022` to use our custom wind plugin. This PR updates the rest of the example worlds included in the repo. Consistent with the wind settings in these example worlds, the wind velocity is set to 0, so the only effect should be to create the two wind speed and direction topics. The real purpose of doing this is to avoid having any confusing code snippets lying around.

To test:

`ros2 launch vrx_gz competition.launch.py world:=<taskname>_task`

In another terminal, confirm wind topics are publishing correctly through the ros bridge:
```
ros2 topic echo /vrx/debug/wind/direction
ros2 topic echo /vrx/debug/wind/speed
```
